DIPs/DIP1NNN-MVF.md

```diff
@@ -0,0 +1,159 @@
+
+## Description
+
+This DIP proposes a very simple path forward that will not introduce any breaking changes if users utilize the transition features being proposed.  Within 2 years, starting from the approval of this DIP, D will be `@safe`-by-default.  Existing users, with the flip of a compiler switch, will be able to maintain existing behavior and avoid any breaking changes.  Using the proposed new `version` identifier, `D_SystemByDefault`, users will be able to precisely manage the transition in their code with the full power of D's exceptional meta-programming facilities.
+
+### Stage 1: Epoch minus 2 years: Raising Awareness
+The approval announcement of this DIP will serve as the official announcement, and the beginning of the transition, marking **Epoch minus 2 years**.
+
+Upon approval of this DIP, a new page will be created at [dlang.org](http://dlang.org) detailing the transition plan, schedule, milestones and any additional information to help users understand when the transition will occur, how it will affect them, and how they can transition with D.  It will be updated and maintained as the transition progresses.
+
+The [dlang.org home page](http://dlang.org) and [dlang.org download page](https://dlang.org/download.html) will feature a prominent notice informing users of the transition, with a link to the aforementioned page describing the transition.  The [dlang.org calendar](https://dlang.org/calendar.html) will be updated with any important milestones, and will be updated and maintained as the transition progresses.
+
+Additional announcements can also be made throught [The D Blog](https://dlang.org/blog/), reddit, Hackker News, and potentially other social media to reach as broad an audience as possible.  This will also serve to raise greater interest in D.
+
+This stage will help users understand what to expect and give them a full year to plan how they wish to transition with D, ask questions, and raise any potential issues before anything happens.
+
+### Stage 2: Epoch minus 1 year: Introduction of Compiler Flags
+
+DMD will be updated with 2 compiler flags, `-transition=safeByDefault` and `-revert=safeByDefault`.
+
+  * Invoking the compiler with `-transition=safeByDefault` will cause the compiler to produce code that is `@safe`-by-default.
+  * Invoking the compiler with `-revert=safeByDefault` will cause the compiler to produce code that is `@system`-by-default, maintaining the status quo.
+  * Invoking the compiler with neither flag will cause the compiler to produce code that is `@system`-by-default while emitting a deprecation warning message informing users of the upcoming transition, where to find more information, and how to silence the warning message by invoking the compiler with either `-transition=safeByDefault` or `-revert=safeByDefault`.  Users are welcome to tolerate the warning message until **Epoch**, at which time the warning message will no longer appear.
```

Comment:
I think it should be specified what happens if both are specified. I think the latest selected option should prevail, so you can set a system-wide default for example, and then override it per-project.

Reply:
Done.  Thank you!